### PR TITLE
backport: for ticket 7199 and 7318

### PR DESCRIPTION
**Pull request by catenacyber**
## Ticket

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/7407
https://redmine.openinfosecfoundation.org/issues/7368

### CODE REVIEW

File: tests/firewall/firewall-06-tls-sni-enforce/test.yaml [hunk 1]
```diff
@@ -51,18 +47,6 @@ checks:
     count: 26
     match:
       event_type: drop
-- filter:
-    min-version: 8
-    count: 0
-    match:
-      event_type: alert
-      pkt_src: "stream (flow timeout)"
```
Comment (jufajardini) [inline tests/firewall/firewall-06-tls-sni-enforce/test.yaml:59]: no value in keeping this check?

Comment (catenacyber) [reply tests/firewall/firewall-06-tls-sni-enforce/test.yaml:59]: It is improved by checking the alerts are all wire/pcap

**Review by jufajardini — APPROVED** [review]
LGTM

**Review by victorjulien — APPROVED** [review]

**Review by jasonish — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #2181, thanks!

**Pull request merged**